<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frameLeft">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>211</width>
      <height>511</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>191</width>
       <height>371</height>
      </rect>
     </property>
     <property name="title">
      <string>图片属性</string>
     </property>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>30</y>
        <width>190</width>
        <height>321</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="labFormat">
           <property name="text">
            <string>图像格式</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="editImg_Format_2"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="labWidth">
           <property name="text">
            <string>图像宽度</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="editImg_Width_2"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="labHeight">
           <property name="text">
            <string>图像高度</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="editImg_Height_2"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="labDepth">
           <property name="text">
            <string>图像深度</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="editImg_Depth_2"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="labBP">
           <property name="text">
            <string>位平面数</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="editImg_BitPlace"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLabel" name="labSize">
           <property name="text">
            <string>图像大小</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="editImg_SizeByte_2"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBSamp">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>0</y>
      <width>561</width>
      <height>511</height>
     </rect>
    </property>
    <property name="title">
     <string>示例图片</string>
    </property>
    <widget class="QScrollArea" name="scrollArea">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>561</width>
       <height>491</height>
      </rect>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="SamplePic">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>559</width>
        <height>489</height>
       </rect>
      </property>
      <widget class="QLabel" name="sampLabel">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>551</width>
         <height>481</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actOpen"/>
   <addaction name="actSave"/>
   <addaction name="separator"/>
   <addaction name="actRanCrop"/>
   <addaction name="actFlip"/>
   <addaction name="actRotate"/>
   <addaction name="actEnhance"/>
   <addaction name="actResize"/>
   <addaction name="actShake"/>
  </widget>
  <action name="actOpen">
   <property name="text">
    <string>打开</string>
   </property>
  </action>
  <action name="actSave">
   <property name="text">
    <string>保存</string>
   </property>
  </action>
  <action name="actRanCrop">
   <property name="text">
    <string>随机裁剪</string>
   </property>
  </action>
  <action name="actFlip">
   <property name="text">
    <string>图片翻转</string>
   </property>
  </action>
  <action name="actRotate">
   <property name="text">
    <string>图片旋转</string>
   </property>
  </action>
  <action name="actEnhance">
   <property name="text">
    <string>亮度增强</string>
   </property>
  </action>
  <action name="actResize">
   <property name="text">
    <string>尺度变化</string>
   </property>
  </action>
  <action name="actShake">
   <property name="text">
    <string>颜色抖动</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
